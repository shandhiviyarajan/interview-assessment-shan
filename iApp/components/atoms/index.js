import { Button } from './Button';
import { InputText } from './InputText';
import { Text } from './Text';
export { Button, InputText, Text };
