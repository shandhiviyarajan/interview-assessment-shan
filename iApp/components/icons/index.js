import React from 'react';

import Icon from 'react-native-vector-icons/Ionicons';
export const AddIcon = () => {
  return <Icon name="add-circle" size={44} color="#000" />;
};
