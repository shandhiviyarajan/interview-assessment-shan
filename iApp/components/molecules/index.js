export { AlertMessage } from './alert';
export { ProductItem } from './productItem';
