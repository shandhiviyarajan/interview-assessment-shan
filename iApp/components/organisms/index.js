//sample file
//sample file
import React from 'react';
function SampleOrganisms() {
  return <></>;
}

export default SampleOrganisms;
