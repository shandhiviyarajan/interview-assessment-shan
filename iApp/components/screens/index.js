export { ForgotPassword, LoginScreen, RegisterScreen } from './auth';
export { ProductsPage, SingleProductPage } from './products';
export { CartPage, CheckoutPage, CheckoutSuccess } from './cart';
export { ProfilePage } from './profile';
