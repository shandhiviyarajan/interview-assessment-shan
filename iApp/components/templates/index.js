//sample file
import React from 'react';
function SampleTemplate() {
  return <></>;
}

export default SampleTemplate;
