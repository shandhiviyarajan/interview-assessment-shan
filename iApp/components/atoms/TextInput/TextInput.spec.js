import React from 'react';
import {render} from '@testing-library/react-native';

import {TextInput} from '../../components';

it('Should render the TextInput component', async () => {
  const {toJSON} = render(<TextInput onChangeText={() => {}} />);
  expect(toJSON()).toMatchSnapshot();
});
