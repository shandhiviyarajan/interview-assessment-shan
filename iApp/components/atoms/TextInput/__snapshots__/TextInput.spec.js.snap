// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the TextInput component 1`] = `
<TextInput
  keyboardType="default"
  onChangeText={[Function]}
  placeholder="Placeholder"
  placeholderTextColor="#ADADAD"
  style={
    Object {
      "backgroundColor": "#EFEFEF",
      "borderColor": "#EFEFEF",
      "borderRadius": 4,
      "borderWidth": 0,
      "color": "#1C1C1C",
      "fontSize": 14,
      "fontStyle": "normal",
      "fontWeight": "500",
      "height": undefined,
      "marginBottom": undefined,
      "marginHorizontal": undefined,
      "marginLeft": undefined,
      "marginRight": undefined,
      "marginVertical": 0,
      "padding": 10,
      "shadowColor": "#000000",
      "width": undefined,
    }
  }
  textAlign="left"
  value=""
/>
`;
